src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+
+        System.out.println(ball1.getRandomBall());
+        System.out.println(ball2.getRandomBall());
+        System.out.println(ball3.getRandomBall());
+
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please don't add redundant empty lines to your code. The empty line after the last statement in the main method is not needed.